src/catalog_state.h

```diff
@@ -2,7 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#pragma once
+#ifndef SRC_BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_CATALOG_STATE_H_
```

Comment:
this should actually just be `CATALOG_STATE_H_` unfortunately. We should create a directory hierarchy for these to fix that

Reply:
👍 I will fix by removing SRC_BRAVE_VENDOR_ right now and then can create another ticket once ready to change the directory hierarchy if you are happy with that?

Reply:
Fixed